helm_deploy/values-prod.yaml

```diff
@@ -4,6 +4,23 @@
 generic-service:
   ingress:
     host: approved-premises-api.hmpps.service.justice.gov.uk
+    tlsSecretName: hmpps-approved-premises-api-prod-cert
+
+  env:
+    SPRING_PROFILES_ACTIVE: prod
+    APPLICATIONINSIGHTS_CONFIGURATION_FILE: applicationinsights.json
+    HMPPS_AUTH_URL: https://sign-in.hmpps.service.justice.gov.uk/auth
+    SERVICES_COMMUNITY-API_BASE-URL: https://community-api-secure.probation.service.justice.gov.uk
+    SERVICES_AP-DELIUS-CONTEXT-API_BASE-URL: https://approved-premises-and-delius.delius.probation.hmpps.dsd.io
+    SERVICES_ASSESS-RISKS-AND-NEEDS-API_BASE-URL: https://assess-risks-and-needs.hmpps.service.justice.gov.uk
+    SERVICES_HMPPS-TIER_BASE-URL: https://hmpps-tier.hmpps.service.justice.gov.uk
+    SERVICES_PRISONS-API_BASE-URL: https://api.prison.service.justice.gov.uk
+    SERVICES_CASE-NOTES_BASE-URL: https://offender-case-notes.service.justice.gov.uk
+    SERVICES_AP-OASYS-CONTEXT-API_BASE-URL: https://approved-premises-and-oasys.hmpps.service.justice.gov.uk
```

Comment:
I think only Community API and Prisons API need valid cred and the others SERVICES can have placeholder values for now as CAS3 won't be asking the API to fetch anything from them.